requirements.txt

```diff
@@ -1,5 +1,5 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
--f https://data.pyg.org/whl/torch-1.12.1+cpu.html  # For torch geometric skip torch-scatter/sparse setup.py
```

Comment:
had to remove comments because new python rules have trouble parsing them :(

Reply:
If pyg still works without it then thats great